engine/src/main/java/org/terasology/rendering/ShaderManagerLwjgl.java

```diff
@@ -122,7 +121,6 @@ public void initShaders() {
         prepareAndStoreShaderProgramInstance("sky", new ShaderParametersEmpty());
         prepareAndStoreShaderProgramInstance("chunk", new ShaderParametersChunk());
         prepareAndStoreShaderProgramInstance("particle", new ShaderParametersParticle());
-        prepareAndStoreShaderProgramInstance("block", new ShaderParametersBlock());
```

Comment:
No, the block shaders aren't useless. However, they are uselss for the rendering operations.

For reference, look at the function `org.terasology.world.block.tiles.WorldAtlasImpl.buildAtlas()` that uses this shader.